game/tasks/task01_S_player.py

```diff
@@ -0,0 +1,190 @@
+"""Task 01: Player.
+
+Welcome to Design Pattern Island! Your first task will be to personalize your character
+by applying the "S" of the S.O.L.I.D design principles.
+
+> (S) **Single Responsibility**
+>
+> A class, function, method or module should have a single responsibility.
+> If it has many responsibilities, it increases the possibility of bugs!
+
+"""
+
+import random
+
+import arcade
+from loguru import logger
+
+from ..core.registration import Register, SpriteRegister
+from ..core.views.rpg_sprites import CharacterSprite, PlayerSprite
+
+"""
+
+Each task can be reloaded when you make code changes with either `Ctrl R` or `CMD R`!
+
+To play the game with reload, both `SOURCE_NAME` and `load_sprites` need to be
+implemented. If these get removed, you will see an error when reloading. After fixing,
+you can reload without needing to quit.
```

Comment:
I think there's a chance the user might misunderstand and think they need to implement `SOURCE_NAME` and `load_sprites`.

Maybe we can rephrase it to something like this?
```
If reload is not working, you may have accidentally removed one of the following from the file:
- `SOURCE_NAME` variable
- `load_sprites` function
```

Reply:
That's a good point! I think this might be a little more clear, but we can fix it up in future commits as well

```suggestion
Avoid making any changes to `SOURCE_NAME` or the function signature to `load_sprites`
because those are required for reload, but you can change the content of the
`load_sprites` function. On reload, the game will attempt to identify any errors and
recommended fixes, so you can often fix and reload without guitting!
```